iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -234,6 +240,77 @@ public class HiveIcebergStorageHandler implements HiveStoragePredicateHandler, H
             }
           };
 
+  private static final transient BiFunction<Integer, Integer, Function<List<ExprNodeDesc>, ExprNodeDesc>>
```

Comment:
this class looks overloaded already, could we move these functions into some helper class?

Reply:
Done.